setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name="newrelic-lambda-cli",
-    version="0.7.6",
+    version="0.7.7",
```

Comment:
If this was at 1.0.0 or higher this change would be a major version bump as it isn't fully backwards compatible (The template is changed and doesn't match the previous). However, since the major version is 0 indicating it's in initial development I'm suggesting this version be 0.8.0 to indicate that this is a large change.
```suggestion
    version="0.8.0",
```